java/security/audit/defaulthttpclient-is-deprecated.java

```diff
@@ -0,0 +1,31 @@
+// cf. https://mkyong.com/java/the-type-defaulthttpclient-is-deprecated/
```

Comment:
you can move this to the message if you want. I have been very generous in including URL-s in my message!